### PR TITLE
check for new datamodel before getting workflows

### PR DESCRIPTION
**Pull request by mikemrm**
Without this check boots is incompatibles with the old version of
cacher. This check skips getting workflows when not using the new
datamodel.

### CODE REVIEW

File: job/job.go [hunk 1]
```diff
@@ -99,22 +100,24 @@ func CreateFromIP(ip net.IP) (Job, error) {
 		return Job{}, err
 	}
 
-	hd := d.Hardware()
-	hwID := hd.HardwareID()
+	if os.Getenv("DATA_MODEL_VERSION") == "1" {
```
Comment (mmlb) [inline job/job.go:103]: How about 
```
if os.Getenv("DATA_MODEL_VERSION != "" {
    return j, nil
}

old new code here
```
instead

Comment (mikemrm) [reply job/job.go:103]: I can do that

Comment (mmlb) [reply job/job.go:103]: IDK what you're talking about, there's definitely a `!= "1"` up there. All the other uses of DATA_MODEL_VERSION have been in a switch statement and here want to do early return instead.

Comment (mikemrm) [reply job/job.go:103]: 🤔  that was definitely something different a second ago, heh. cool, will do.

**Pull request merged**